cookiecutter.json

```diff
@@ -2,14 +2,16 @@
     "full_name": "Jørgen S. Dokken",
     "email": "dokken@simula.no",
     "github_username": "jorgensd",
-    "project_name": "Package Name",
-    "module_name": "{{ cookiecutter.project_name.lower().replace(' ', '_').replace('-', '_') }}",
+    "project_name": "Research Project 1",
+    "repository_name": "{{ cookiecutter.project_name.lower().replace(' ', '_').replace('-', '_') }}",
+    "module_name": "{{cookiecutter.repository_name}}",
     "project_short_description": "Tools ensuring reproducible and standarized Python packaging.",
     "version": "0.1.0",
     "linting": "y",
     "testing": ["pytest", "pytest-cov", "none"],
     "use_argparse": "y",
     "build_docs": "y",
```

Comment:
I also think we should add a better description here (what does `build_docs` mean?) .

Reply:
Jupyter_book maybe?